2-promises/solution.js

```diff
@@ -23,17 +23,29 @@ const id = yourRandomMethod() //third run
 6. use promise chaining accordingly, don't forget error handling
 7. log the resultant fullname, or the error, at the end
 */
+const firstNameMethodImport = require("./firstnames")
+const lastNameMethodImport = require("./lastnames")
```

Comment:
I believe I suggested avoiding using the string itself `Method` within or for a method/function  reference or name. You should also set a descriptive/meaningful method name 
```suggestion
const getFirstName = require("./firstnames")
const getLastName = require("./lastnames")
```